src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoMachine {
+
+    private final List<Integer> winNumbers;
+    private final Integer bonusNumber;
+
+    public LottoMachine(final List<Integer> winNumbers, final int bonusNumber) {
+        this.winNumbers = winNumbers;
+        this.bonusNumber = bonusNumber;
+    }
+
+    public boolean isContain(final int number) {
+        return winNumbers.contains(number);
+    }
+
+    public boolean isEqualBonusNumber(final int number) {
+        return bonusNumber.equals(number);
```

Comment:
int에 대한 비교는 `==`로해주면 좋을 것 같아요! [참고](https://stackoverflow.com/questions/29139274/how-equal-operator-works-with-primitive-and-object-type-data)

```suggestion
        return bonusNumber == number;
```